src/main/java/uk/gov/hmcts/reform/sscs/model/single/hearing/HearingResponse.java

```diff
@@ -21,4 +24,22 @@
     private LocalDateTime timeStamp;
 
     private Number versionNumber;
+
+    private String listAssistTransactionID;
+
+    private LocalDateTime receivedDateTime;
+
+    private Integer responseVersion;
+
+    @JsonProperty("laCaseStatus")
+    @EnumPattern(enumClass = ListingCaseStatus.class, fieldName = "listingCaseStatus")
```

Comment:
Should these two be enums?

Reply:
in my mapping, they are enum
![image](https://user-images.githubusercontent.com/93123095/159455984-b091b5f5-18a8-43b4-8e19-c315d1fca287.png)

Reply:
I might be missing something but I was asking why they are String types when they seem to be pointing at an Enum type? Shouldn't it be private ListingCaseStatus listingCaseStatus?

Reply:
I see other enum in the repo are use the same way, so I just follow. 
I think without using a enum will allow a bit of flexible that the data other than the enum?